### PR TITLE
Add markdown component

### PR DESCRIPTION
**Pull request by rtkpghuman**
## Testing:
manual

## Screenshots:
Screen Shot 2020-11-10 at 12 13 24 PM

### CODE REVIEW

File: src/components/markdown/Markdown.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import * as MarkdownIt from 'markdown-it';
+
+type Props = {
+  children?: React.ReactNode;
+};
+
+/*
+  Adding this as a simpler way to manage markdown wrt to the current
+  implementations found in MarkdownRenderer this implementation doesnt
```
Comment (rtktpeng) [inline src/components/markdown/Markdown.tsx:10]: Since this is an open source public repo, I'd leave this reference out.

**Review by rtktpeng — COMMENTED** [review]
are we missing an export from the index file?

**Comment by rtktpeng** [comment]
Similar reminder to avoid referencing internal resources in PR title and description. 🙂

File: src/components/markdown/__tests__/Markdown.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+import { mount, shallow } from 'enzyme';
```
Comment (rtkaaho) [inline src/components/markdown/__tests__/Markdown.spec.tsx:3]: no need for mount here

**Review by rtkaaho — APPROVED** [review]
🏃

**Pull request merged**